### PR TITLE
Update sbt/g8 version

### PR DESCRIPTION
**Pull request by amaya382**
* sbt: 0.13.16 for g8
* ~sbt: 1.1.0 in template~ (#19)
* giter8: 0.10.0

`g8 file://scala-seed.g8 --name=test` and then `sbt run` works fine with g8 0.10.0 and sbt 1.0.0

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=1.0.0
```
Comment (xuwei-k) [inline project/build.properties]: g8 does not support sbt 1.0 https://github.com/foundweekends/giter8/issues/318

**Review by xuwei-k — CHANGES_REQUESTED** [review]

File: src/main/g8/project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=0.13.16
+sbt.version=1.0.0
```
Comment (seblm) [inline src/main/g8/project/build.properties]: last sbt version is `1.0.2`

**Review by seblm — CHANGES_REQUESTED** [review]

**Comment by SethTisue** [comment]
I've just merged #19 which updates the sbt version.

@amaya382 I'm sorry, but this PR got lost in the shuffle. there is still a piece of this PR which #19 didn't cover, namely the sbt-giter8 version bump. if you rebase it, I'll merge it.

**Comment by amaya382** [comment]
@SethTisue I rebased and updated it to the latest version

**Comment by SethTisue** [comment]
thank you!

**Pull request merged**